setup.py

```diff
@@ -105,6 +107,7 @@ def get_hierarchy():
                      include_dirs=[LIKWID_INCPATH],
                      libraries=[LIKWID_LIB],
                      library_dirs=[LIKWID_LIBPATH],
+                     define_macros=[("LIKWID_MAJOR", major), ("LIKWID_MINOR", minor), (("LIKWID_RELEASE", release))],
```

Comment:
Does this work? The variables `major`, `minor` and `release` are defined in a `try` block, so they are probably not in scope anymore.

Reply:
There exists a function `likwidversion` that returns the version at runtime.

Reply:
Yes, but the MACROS are not defined. It seems like there's a differently named MACRO in likwid.h, which comprises a string of the full version, .e.g., "5.3.2". So it seems like the function is broken when not defining the macros here